02-RTB/verbali-interni/2024-11-04.typ

```diff
@@ -0,0 +1,249 @@
+#import "/lib/verbale.typ": *
+
+#show: body => verbale(
+  data: datetime(day: 04, month: 11, year: 2024),
+  tipo: [interno],
+  versioni: (
+    (
+      vers: "0.1.0",
+      date: datetime(day: 05, month: 11, year: 2024),
+      autore: p.matteo,
+      //verifica: SPRITZ{INSERISCI_IL_VERIFICATORE},
+      descr: "Redazione del contenuto",
+    ),
+  ),
+  stato: [Redatto],
+  presenze: (p.emanuele, p.loris, p.marco, p.matteo, p.samuele, p.sara),
+  odg: [Quinta riunione di ALimitedGroup: vengono analizzate le osservazioni nel documento di aggiudicazione degli appalti per procedere a migliorare il nostro Way of Working],
+  disambiguatore: [],
+  body,
+)
+
+= Informazioni generali
+
+#inizio-verbale-interno([*virtuale*], [*4 Novembre 2024*], [*15.00*], [*17.35*])
+
++ Decisioni in merito alla rotazione dei ruoli
++ Migliorie alla parte pubblica del repository
++ Importanti cambiamenti nella redazione dei verbali
++ Recupero delle precedenti decisioni e attività
++ Inizio redazioni di ulteriori documenti
++ Assegnazione dei ruoli sino al giorno 11-11-2024
+
+#pagebreak()
+= Dettaglio riunione
+== Decisioni in merito alla rotazione dei ruoli
+
+Il gruppo _ALimitedGroup_ ha accolto la problematica riguardante le mancate indicazioni in merito alla rotazione dei ruoli, non fornite nei documenti di candidatura.
+
+I componenti hanno compreso l'importanza di tale operazione e hanno stabilito che, almeno in questa fase iniziale segnata dall'inizio vero e proprio del progetto, i ruoli subiranno un cambiamento ogni *circa 7 giorni*.\
+Il gruppo concorda però nell'affermare che tale periodo di tempo non può che essere solo che momentaneo e con lo scopo di produrre efficacemente la prima documentazione di base, volta principalmente a sanare le problematiche rilevate durante l'aggiudicazione degli appalti.
+
+Il gruppo prevede che nel prossimo futuro il cambiamento dei ruoli *non potrà sicuramente avvenire prima di 14 giorni*, minimo periodo per produrre avanzamenti concreti, impossibili da realizzare con un periodo di tempo più corto.
+
+#pagebreak()
+
+== Migliorie alla parte pubblica del repository
+
+Il gruppo accoglie il suggerimento riguardante l'apertura delle risorse in una scheda dedicata: decide perciò di modificare la parte pubblica del repository (sito web) quanto prima per ottenere questo effetto.
+
+Per migliorare ulteriormente l'organizzazione del repository, stabilisce altresì di inserire tutti i nuovi documenti prodotti dal giorno 04-11-2024 sino al superamento della Requirements and Tecnology Baseline (RTB) in una sezione appositamente realizzata: la parte pubblica del repository (sito web) dovrà riflettere questo cambiamento.
```

Comment:
In questo paragrafo due parole in inglese vanno in corsivo: Requirements and Tecnology Baseline, repository